cpp/vm/evmzero/evmzero.cc

```diff
@@ -23,6 +22,11 @@
 #include "vm/evmzero/profiler.h"
 #include "vm/evmzero/sha3_cache.h"
 
+#if defined(TOSCA_COVERAGE)
+// Since Gcc 11, before __gcov_flush
+extern "C" void __gcov_dump();
```

Comment:
There might be a merge issue here. This code is part of the code coverage PR for C++ code and should not be included in the PR adding Go linter support.

Reply:
sorry about that, pretty bad rebase.